### DISCUSSION TITLE
Migration from version 3 to 4

### DISCUSSION THREAD

**Discussion by gpsaliola** [Mapsui/Mapsui#2083] [Q&A] [answered]
I can't find Mapsui.Geometries.Polygon match in version 4.
I expected to find something like Mapsui.MPolygon but it doesn't exist.
I didn't find anything in https://mapsui.com/documentation/v4.0-upgrade-guide.html.
Can you kindly give me some suggestions?
Thank you

**Comment by pauldendulk** [comment, marked as answer]
This was a big change in going to v4. Everything in Mapsui.Geometries was replaced with an external library called [NetTopologySuite (NTS)](https://github.com/NetTopologySuite/NetTopologySuite). To use is you need to add the Mapsui.Nts nuget package. In the guide this is described with 

> Mapsui.Nts, which has a dependency on NetTopologySuite (NTS). It replaces everything that was in Mapsui.Geometries.

There is a sample category called 'Geometries' you may want to look into.

**Comment by gpsaliola** [comment]
Thank you! I'll try it.
